packages/web/components/Form/FormWizard.js

```diff
@@ -153,12 +153,10 @@ const FormWizard = ({ steps, currentStep, onSubmit, onPrev }) => {
 	const prevStep = currentStepIndex === 0 ? false : steps[currentStepIndex - 1].slug;
 
 	const handleSubmit = (data) => {
-		window.scrollTo({ top: 0 });
 		onSubmit({ data, step: currentStepSlug, nextStep });
 	};
 
 	const handlePrev = () => {
-		window.scrollTo({ top: 0 });
```

Comment:
Essa parte aqui foi retirada por algum motivo? Isso aqui é para não acontecer aquela questão de o usuário mudar de página no mobile e a página não scrollar para o início...foi até foi que percebeu, lembra?

Reply:
Eu lembro, é que no desktop não tá funcionando muito bem (a tela dá um pulo e quebra o fluxo).... Acho que podemos criar uma issue pra lidar com isso só no mobile.

Reply:
Vou reativar aqui e depois agente vê...